src/v/cluster/topics_frontend.cc

```diff
@@ -269,6 +307,16 @@ ss::future<std::vector<topic_result>> topics_frontend::update_topic_properties(
 
         auto results = co_await ssx::parallel_transform(
           std::move(updates), [this, timeout](topic_properties_update update) {
+              if (_features.local().should_sanction()) {
```

Comment:
The same question about internal topics that @IoannisRP highlighted applies here as well. We might want to allow updates to internal topics and perhaps even some "well-known" kafka-namespace topics.

Reply:
I'll add `is_user_topic(update.tp_ns)`